MangoPay/PayOut.php

```diff
@@ -19,12 +33,24 @@ class PayOut extends Transaction
      */
     public $PaymentType;
 
+    /**
+     * One of PayOutPaymentDetails implementations, depending on $PaymentType
+     * @var PayOutPaymentDetailsBankWire
```

Comment:
```suggestion
     * @var object
```
This has different types